cogs/wordfilter/wordfilter.py

```diff
@@ -64,6 +66,12 @@ async def regex(self, ctx):
         out messages.
         """
 
+    @wordFilter.group(name="stat", aliases=["st"])
+    async def stat(self, ctx):
+        """
+        access censorship statistics
+        """
```

Comment:
nit: Put on one line and capitalize first letter since this is visible from helptext:
```suggestion
        """Access filter statistics"""
```